site/it/about/bib.md

```diff
@@ -0,0 +1,131 @@
+# White Papers su TensorFlow
+
+Questo documento identifica i white papers su TensorFlow.
+
+## Machine Learning su Larga Scala su Sistemi Distribuiti Eterogenei
+
+[Access this white paper.](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/45166.pdf)
+
+**Abstract:** TensorFlow è un'interfaccia per descrivere algoritmi 
+ed implementazioni di machine learning ed un'implementazione per
+eseguire questi algoritmi.
+Una computazione descritta con TensorFlow può essere eseguita
+senza o con minimi cambiamenti su una grande varietà di sistemi
+eterogenei, dai dispositivi mobili come telefoni e tablet
+fino a grandi sistemi distribuiti di centinaia di macchine
+e migliaia di dispositivi di calcolo come le schede GPU.
+Il sistema è flessibile e può essere usato per esprimere una 
```

Comment:
Here you are using esprimere. So I suppose you can align with the intital interpretation